frontend/sass/pages/_admin.scss

```diff
@@ -196,3 +196,111 @@
     }
   }
 }
+
+.admin-panel-status-suspended {
+  color: map.get(map.get(abstracts.$highlight-colors, "pink"), "default");
+  margin-left: 5px;
+}
+
+.admin-panel-status-active {
+  color: map.get(map.get(abstracts.$highlight-colors, "cyan"), "default");
+  margin-left: 5px;
+}
+
+.admin-panel-report-label {
+  display: none;
+}
+
+.admin-panel-reports-grid {
+  min-width: fit-content;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 300px 1fr;
+  align-items: center;
+  justify-content: space-evenly;
+  padding: 10px 0px;
+  border-bottom: 1px solid;
+  border-color: abstracts.$primary-darkgrey;
+  & > * {
+    margin-left: 5%;
+    margin-right: 5%;
+  }
+
+  a {
+    margin-left: 5px;
+  }
+  input {
+    margin-left: 5%;
+  }
+}
+
+.admin-panel-section-header {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 300px 1fr;
+  min-width: fit-content;
+  justify-content: space-around;
+
+  h4 {
+    min-width: fit-content;
+    margin-left: 5%;
+    margin-right: 5%;
+  }
+}
+.admin-panel-report-content {
+  word-wrap: break-word;
+  max-width: 300px;
+}
+
+.admin-panel-reports {
+  @media only screen and (max-width: abstracts.$large-width) {
+    .admin-panel-reports-grid {
+      grid-template-columns: repeat(3, minmax(0, 1fr)) max-content;
```

Comment:
Tässä voisi olla vaan `repeat(4, minmax(0, 1fr))`, niin kaikki eri osiot pysyisi tasassa, nyt Target venyttelee näitä vähän miten sattuu kun siinä voi olla monenmittaista sisältöä riippuen onko suspended vai ei.